eventserver/server_billable_handler.go

```diff
@@ -26,43 +30,120 @@ func BillableEventsHandler(store eventio.BillableEventReader, uaa auth.Authentic
 			return echo.NewHTTPError(http.StatusBadRequest, err)
 		}
 		// query the store
-		rows, err := store.GetBillableEventRows(filter)
+
+		rowOfRows := RowOfRows{}
+		defer rowOfRows.Close()
+
+		months, err := filter.SplitByMonth()
 		if err != nil {
 			return err
 		}
-		defer rows.Close()
-		// stream response to client
-		c.Response().Header().Set(echo.HeaderContentType, echo.MIMEApplicationJSONCharsetUTF8)
-		c.Response().WriteHeader(http.StatusOK)
-		if _, err := c.Response().Write([]byte("[\n")); err != nil {
-			return err
-		}
-		c.Response().Flush()
-		next := rows.Next()
-		delim := "\n"
-		for next {
-			b, err := rows.EventJSON()
+		for _, monthFilter := range months {
+			ok, err := consolidatedStore.IsRangeConsolidated(monthFilter)
```

Comment:
Consider renaming `ok` here. Maybe `isRangeConsolidated`? This isn't a good/bad situation

Reply:
done